src/AutoRest.CSharp/LowLevel/AutoRest/LowLevelOutputLibrary.cs

```diff
@@ -18,16 +18,20 @@ internal class LowLevelOutputLibrary : OutputLibrary
         private readonly CodeModel _codeModel;
         private readonly BuildContext<LowLevelOutputLibrary> _context;
         private CachedDictionary<OperationGroup, LowLevelRestClient> _restClients;
+        private CachedDictionary<OperationGroup, LowLevelDataPlaneClient> _clients;
 
         public LowLevelOutputLibrary(CodeModel codeModel, BuildContext<LowLevelOutputLibrary> context) : base(codeModel, context)
         {
             _codeModel = codeModel;
             _context = context;
             _restClients = new CachedDictionary<OperationGroup, LowLevelRestClient>(EnsureRestClients);
+            _clients = new CachedDictionary<OperationGroup, LowLevelDataPlaneClient>(EnsureClients);
```

Comment:
We need better names for these two fields.

Reply:
Do you have any thoughts on better names?  I might be too close to the problem here to come up with some other terms.  This mirrors the "Clients" and "RestClients" design we have in `DataPlaneOutputLibrary.cs`.

Would `_internalRestClients` and `_publicClients` be better here (and would give a clearer indication that one of these is internal machinery to the library and the other is about something we publicly expose to clients)?

Reply:
Yes, that would probably be better names.

Reply:
Adopted these names.